src/js/game/LevelMVC/LevelView.js

```diff
@@ -1095,7 +1097,7 @@ module.exports = class LevelView {
       sprite.sortOrder = this.yToIndex(destroyPosition[1]) + 2;
     }
 
-    if (this.controller.getIsDirectPlayerControl()) {
+    if (completionHandler && this.controller.getIsDirectPlayerControl()) {
```

Comment:
be careful about addition conditions to `if`s with an `else`; I believe in this scenario we want to avoid the `else` case when we're in direct control mode, so the `if (completionHandler)` should just go inside the block